service/manager.go

```diff
@@ -45,6 +42,10 @@ type ExitCallback func(shutdown Exit) int
 // Implements Manager interface
 type manager struct {
 	serviceCore
+
+	StartTimeout time.Duration `yaml:"start_timeout" default:"10s"`
```

Comment:
we seem to be using a mix of underscores and camel case in yaml files. we should settle on one to be consistent.